extensions/hp-extension/src/index.tsx

```diff
@@ -23,14 +24,16 @@ export default {
   preRegistration: ({
     servicesManager,
   }) => {
-    const { HangingProtocolService } = servicesManager.services;
-    HangingProtocolService.addCustomViewportSetting('zoomPan', "Set initial zoom, WITH camera event", 
-      setViewportZoomPan, 
-      storeViewportZoomPan);
+    const { HangingProtocolService, SyncGroupService } = servicesManager.services;
+    // HangingProtocolService.addCustomViewportOption('zoomPan', "Set initial zoom, WITH camera event", 
+    //   setViewportZoomPan, 
+    //   storeViewportZoomPan);
     HangingProtocolService.addCustomAttribute('ViewCode', 'View Code Designator:Value', viewCodeAttribute);
     HangingProtocolService.addCustomAttribute('Laterality', 'Laterality of object', lateralityAttribute);
     HangingProtocolService.addCustomAttribute('seriesDescriptions', 'Series Descriptions', seriesDescriptionsFromDisplaySets);
     HangingProtocolService.addCustomAttribute('numberOfDisplaySets', 'Number of displays sets', numberOfDisplaySets);
+  
+    SyncGroupService.setSynchronizer('initialzoompan', initialZoomPan);
```

Comment:
It seems there is a very specific version, that exposes servicesManager.services.SyncGroupService. setSynchronizer.
We again come to the coupling thing here. I was wondering if we could at least have a conditional to ensure there is this  method.